### ISSUE TITLE
Data Virtualization (Extension) - Unsupported platform: darwin

### ISSUE THREAD

**Issue by jrdodds** [microsoft/azuredatastudio#24728]
Type: <b>Bug</b>

Updated from Intel to Apple Silicon build of Azure Data Studio for macOS and the Data Virtualization extension began failing with an error:

```
Failed to start Scale Out Data service error:
Unsupported platform: darwin

Source: Data Virtualization (Extension)
```

(This is not a blocking issue for me but it may be for someone else.)

Extension version: 1.12.0
Azure Data Studio version: azuredatastudio 1.46.1 (ba29842b81dec01177415e53948ca2168e69c3f8, 2023-10-02T18:15:27.480Z)
OS version: Darwin arm64 21.6.0
Restricted Mode: No
Preview Features: Enabled
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Pro (10 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|32.00GB (0.70GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>

**Comment by Charles-Gagnon** [comment]
@jrdodds It looks like you're using an outdated version of the extension - could you try updating to the latest (1.13.2) and see if you still get this error?

This version will still be using the Intel version of the binaries though, but should be runnable. We'll follow up with adding the ARM64 version for the next release.

**Comment by jrdodds** [comment]
The issue appears to be resolved with v1.13.2. Thanks
